### ISSUE TITLE
Support for OpenPower computers

### ISSUE THREAD

**Issue by GreenmanPGI** [cryptoadvance/specter-desktop#1139]
Can you please offer support for OpenPower computers. They are the only secure computers on the market since they contain no proprietary code (even at the CPU level). For more information: https://www.raptorcs.com/content/base/faq.html

**Comment by k9ert** [comment]
Does OpenPower support python? What's missing?

**Comment by GreenmanPGI** [comment]
Yes OpenPower supports python but the AppImage will not work on OpenPower. So I don't know to install / run the program on an OpenPower Machine.
